resources/views/components/listing.blade.php

```diff
@@ -1,38 +1,33 @@
-@props(['query'])
+@props(['rootPath' => null])
 
 @pushOnce('head', 'es_url-preconnect')
     <link rel="preconnect" href="{{ config('rapidez.es_url') }}">
-
-    @if ($file = vite_filename_path('Listing.vue'))
-        @vite([$file])
-    @endif
+    @vite(vite_filename_paths(['Listing.vue', 'InstantSearch']))
 @endPushOnce
 
 <div class="min-h-screen">
     <listing
-        :additional-filters="{!! isset($query) ? "['query-filter', 'category', 'score-position']" : "['category', 'score-position']" !!}"
+        {{ $attributes }}
         :additional-sorting="[{
             label: window.config.translations.newest,
-            dataField: 'created_at',
-            sortBy: 'desc'
+            field: 'created_at',
+            order: 'desc',
+            {{-- TODO: Extract this somewhere? --}}
+            value: config.index_prefix + '_products_' + config.store + '_created_at_desc',
+            key: '_created_at_desc'
         }]"
+        {{-- TODO: Extract this somewhere? --}}
+        :index="config.index_prefix + '_product_' + config.store"
+        inline-template
```

Comment:
What's the alternative for this as `inline-template` isn't available anymore in Vue 3? @indykoning